helm/values.yaml

```diff
@@ -142,6 +142,10 @@ policy:
 # interrupting operation.
 detectionMode: false
 
+# validation mode allows advanced configuration of image validation such as whether to mutate image references
+validationMode:
+  mutateImage: true  # mutate image references to signed digests ('true', default) or keep original reference ('false')
```

Comment:
Would add a disclaimer that this actually lowers security (significantly I would argue, since the threat model for Connaisseur basically is that the attacker has control over the registry or connection to registry).

Reply:
there is a disclaimer in the docs, you think it must be in the helm as well?

Reply:
It basically undermines every security guarantee that Conny provides. So yes. I'm also not sure whether it's something we should allow in the first place, because it so significantly undermines it, but if so, there should be as disclaimers at every point this is mentioned

Reply:
Thinking about it, I would prefix the config var with `unsafe`

Reply:
like `unsafeOnlyValidateTags: false`

Reply:
I disagree in that it undermines EVERY security guarantee. It significantly complicates a practical attack.
Moreover, `detectionMode` is in some ways worse than that as it gives a warning but that may never be read and we do not denote it as `unsafe`

Reply:
also `namespacedValidation` can have the same effect

Reply:
detectionMode on the other hand has a very clear interface. It "detects" but doesn't do more. It has very clear applications for testing and thus can improve security by enabling to get the tool into a pre-existing infrastructure without breakage.

validationMode on the other hand doesn't have a super clear-cut interface. It only validates, but somewhat wants to provide security. The distinction is not apparent from naming alone that it will validate existence of a digest, but make no pull guarantees. While it does enable some infrastructures to validate digests, it doesn't give them the actual guarantees that one would assume to get from an admission controller such as Conny

If you feel strongly about not adding the `unsafe` prefix, how about something like

```
security-enforcement (kinda bad name, let's pick another):
  - manifest-security: true # whether to enforce that the manifest corresponds to a validly signed (not signed, because could be allowlisted but you know i mean) (basically this = false equivalent to detectionMode -> unify with next major)
  - runtime-security: true # whether to enforce the pulled image is actually the signed (not signed, because could be allowlisted but you know i mean) image and not another one (this = false is your validation only mode)
```
(if you find the general idea reasonable, let's sit a few minutes for proper names)